tests/e2e/utils/KubernetesCommandLineToolsExecutor.ts

```diff
@@ -89,6 +94,27 @@ export class KubernetesCommandLineToolsExecutor extends ShellExecutor {
         this.execWithLog(`${this.KUBERNETES_COMMAND_LINE_TOOL} create namespace ${this.namespace}`);
     }
 
+    createProject(projectName: string, timeout: number = TimeoutConstants.TS_SELENIUM_TERMINAL_DEFAULT_TIMEOUT * 2): void {
+        Logger.debug(`${this.getLoggingName(this.createProject.name)}: Create new project "${projectName}".`);
+        const polling: number = TestConstants.TS_SELENIUM_DEFAULT_POLLING;
+        const attempts: number = Math.ceil(timeout / polling);
+        let response: ShellString;
+        for (let i: number = 0; i < attempts; i++) {
+            response = this.execWithLog(`${this.KUBERNETES_COMMAND_LINE_TOOL} new-project ${projectName} -n ${this.namespace}`);
+            if ((response.stderr + response.stdout).includes('already exist')) {
+                Logger.trace(`${this.getLoggingName(this.createProject.name)} - Project already exist #${(i + 1)}, retrying with ${polling}ms timeout`);
```

Comment:
It's interesting to know the case when it's needed to retry creation of already existed project.

Reply:
I see.

In this case there is no need to create project again,if I understood it correctly.
Therefore next single command should work as well, because will not fail in case if project has been already existed (with ending `|| true`):
```
exec(`${this.KUBERNETES_COMMAND_LINE_TOOL} new-project ${projectName} -n ${this.namespace} || true`)
```

Reply:
We need to recreate it, because import from git in DevConsole will be unsuccessful

Reply:
Basically, recreation of project is not a trivial and risky operation, especially if there have workspaces been already created [1].
It would simpler to fail the test if there is project existed, IMHO
@musienko-maxim : what do you think?

https://docs.openshift.com/container-platform/4.10/web_console/odc-about-web-terminal.html#deleting-the-devworkspace-operator-dependency

Reply:
but how should I clean it?

Reply:
okay, lets remove it and checks how stable will be test runs

Reply:
done

Reply:
thanks